libs/partners/pinecone/pyproject.toml

```diff
@@ -21,15 +21,12 @@ disallow_untyped_defs = "True"
 [tool.poetry.dependencies]
 python = ">=3.9,<3.13"
 langchain-core = "^0.3.15"
-pinecone-client = "^5.0.0"
+pinecone = ">=5.4.0"
```

Comment:
This should be `"^5.4.0"`. `>=5.4.0` would include future breaking changes which may not be compatible.

Poetry docs have a nice explanation of how these various syntax elements in version numbers are interpreted. https://python-poetry.org/docs/dependency-specification/

Reply:
Ah thank you so much!